events.html

```diff
@@ -49,6 +49,7 @@
 				<li><a href="https://members.realdevsquad.com/">Members</a></li>
 				<li><a href="https://crypto.realdevsquad.com/">Crypto</a></li>
 				<li><a href="https://website-emberjs.vercel.app/">EmberJS Site</a></li>
+        <li><a href="https://status.realdevsquad.com/">Status</a></li>
```

Comment:
Can we please have this line indented properly?

Reply:
Ok I'll fix it!